src/screens/mentee-home/mentee-home.tsx

```diff
@@ -80,7 +63,7 @@ const MenteeHome: React.FC = () => {
             onMarkRead={handleMarkRead}
           />
           <OpportunitiesSection
-            opportunities={opportunities}
+            opportunities={(opportunities ?? []).slice(0, 3)}
```

Comment:
What is this magic number? Please, move it to a constant. Also try to avoid array manipulation on the go. Do this beforehand